agent/agent.go

```diff
@@ -980,6 +980,12 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string,
 		grip.Error(errors.Wrap(tc.logger.Flush(flushCtx), "flushing logs"))
 	}
 
+	err := a.upsertCheckRun(ctx, tc)
+	if err != nil {
+		grip.Error(errors.Wrap(err, "upserting checkrun"))
+	}
+	grip.Info("Successfully upserted checkRun.")
```

Comment:
grip.Info just goes to the agent logs i think. Should use tc.logger.Task().Info if you want this to go to users.